sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextAnalyticsRequestOptions.java

```diff
@@ -48,6 +49,7 @@ public boolean isIncludeStatistics() {
      *
```

Comment:
for includeStatistics, we should also add a comment about how this information is only available for batch operations so we don't confuse the user when trying it to use in the single document operations